### PR TITLE
Fix rotated pole projections

### PR DESCRIPTION
**Pull request by maartenplieger**

### CODE REVIEW

File: CCDFDataModel/CProj4ToCF.cpp [hunk 1]
```diff
@@ -167,13 +167,9 @@ void CProj4ToCF::initStereoGraphic(CDF::Variable *projectionVariable, std::vecto
   v = getProj4ValueF("y", projKVPList, 0);
   projectionVariable->addAttribute(new CDF::Attribute("false_northing", CDF_FLOAT, &v, 1));
 
-  int projectionUnits = getProjectionUnits(projectionVariable);
```
Comment (lukas-phaf) [inline CCDFDataModel/CProj4ToCF.cpp:170]: The `CProj4ToCF::setProjectionUnits()` is used in a couple of places, but `CProj4ToCF::getProjectionUnits` is only used in the `setProjectionUnits()` function, as far as I can see (once you have removed this one). Can we removed the whole get/set and corresponding defines?

File: CCDFDataModel/CProj4ToCF.cpp [hunk 2]
```diff
@@ -577,8 +573,8 @@ int CProj4ToCF::convertCFToProj(CDF::Variable *projectionVariable, CT::string *p
       } catch (int e) {
       };
 
-      proj4String->print("+proj=geos +lon_0=%f +lat_0=%f +h=%f +a=%f +b=%f", longitude_of_projection_origin.toDouble(), latitude_of_projection_origin.toDouble(), 35807.414063,
-                         semi_major_axis.toDouble() / 1000, semi_minor_axis.toDouble() / 1000);
+      proj4String->print("+proj=geos +lon_0=%f +lat_0=%f +h=%f +a=%f +b=%f +units=km", longitude_of_projection_origin.toDouble(), latitude_of_projection_origin.toDouble(), 35807.414063,
```
Comment (maartenplieger) [inline CCDFDataModel/CProj4ToCF.cpp]: Should be m

File: CCDFDataModel/CProj4ToCF.cpp [hunk 3]
```diff
@@ -814,9 +810,10 @@ int CProj4ToCF::convertCFToProj(CDF::Variable *projectionVariable, CT::string *p
         projectionVariable->getAttribute("false_northing")->getDataAsString(&false_northing);
       } catch (int e) {
       };
-      proj4String->print("+proj=ob_tran +o_proj=longlat +lon_0=%f +o_lat_p=%f +o_lon_p=%f +a=%f +b=%f +x_0=%f +y_0=%f +no_defs", grid_north_pole_longitude.toDouble() + 180.0f,
-                         grid_north_pole_latitude.toDouble(), north_pole_grid_longitude.toDouble(), semi_major_axis.toDouble() / 1000.0f, semi_minor_axis.toDouble() / 1000.0f,
-                         false_easting.toDouble(), false_northing.toDouble());
+      proj4String->print("+proj=ob_tran +o_proj=longlat +lon_0=%f +o_lat_p=%f +o_lon_p=%f +a=%f +b=%f +x_0=%f +y_0=%f +units=km +no_defs", grid_north_pole_longitude.toDouble() + 180.0f,
```
Comment (maartenplieger) [inline CCDFDataModel/CProj4ToCF.cpp]: Should be m

File: CCDFDataModel/CProj4ToCF.cpp (hunk 1 above)
Comment (maartenplieger) [reply CCDFDataModel/CProj4ToCF.cpp:170]: Yes, done!

File: CCDFDataModel/CProj4ToCF.cpp (hunk 2 above)
Comment (maartenplieger) [reply CCDFDataModel/CProj4ToCF.cpp]: Remove units altogether.

File: CCDFDataModel/CProj4ToCF.cpp (hunk 3 above)
Comment (maartenplieger) [reply CCDFDataModel/CProj4ToCF.cpp]: Remove units altogether.

**Review by lukas-phaf — APPROVED** [review]

**Pull request merged**